pkg/logs/fakelog.go

```diff
@@ -0,0 +1,86 @@
+package logs
```

Comment:
As it is only for testing purpose, should this file under a package in `pkg/test` ? :thinking:

Reply:
good question :), just wondering if a test should hold reusable things, while this `fakelogs` specifically for logs only, I felt its better to be close to the package :)

WDYT?

Reply:
ok let's discuss that later, ain't a big deal :wink: